src/connectors/aws_config.py

```diff
@@ -175,12 +176,13 @@ def finalize(connection_name):
 
     config_ingest_task = f'''
 INSERT INTO {landing_table} (
-  raw, hash_raw, event_time, account_id, aws_region, resource_type, arn, availability_zone,
-  resource_creation_time, resource_name, resource_Id, relationships, configuration, tags
+  raw, hash_raw, event_time, configuration_item_capture_time, account_id, aws_region, resource_type, arn,
+  availability_zone, resource_creation_time, resource_name, resource_Id, relationships, configuration, tags
 )
 SELECT value raw
     , HASH(value) hash_raw
-    , value:configurationItemCaptureTime::TIMESTAMP_LTZ(9) event_time
+    , REGEXP_REPLACE(filename, '.+(\\\\d{{4}})(\\\\d{{2}})(\\\\d{{2}})T(\\\\d{{2}})(\\\\d{{2}})(\\\\d{{2}})Z.*', '\\\\1-\\\\2-\\\\3T\\\\4:\\\\5:\\\\6Z')::timestamp_ltz event_time
```

Comment:
what happens if the regex doesn't match?

Reply:
regexp_replace returns the original subject if there's no match, so the statement would fail when it gets a bad timestamp.